### ISSUE TITLE
Declarative Overscroll feedback

### ISSUE THREAD

**Issue by davidsgrogan** [w3c/aria#2785]
@vmpstr and @flackr are working on https://open-ui.org/components/overscroll-actions.explainer/ . They are looking for feedback on the accessibility issues and proposals below, in order of priority.

**1. Container Inertness**
* We plan to inert the overscroll element when it is hidden and inert the main container's content when the overscroll is shown.  [implicitly? can authors override this?]
* We want to know if ARIA is okay with this inert approach, as it differs from `<dialog>` which inerts the entire page. [(old discussion of dialog and island of inertness)](https://github.com/openui/open-ui/issues/741#issuecomment-1544264799#:~:text=then%20through%20some%20error%20of%20css%20then%20the%20entire%20page%20is%20unusable)
* If a control button toggles the area, does it need an exception so it is never inert, or do we need a specific focus management strategy?

**2. Navigation**
* Should arrow keys or the mouse wheel access the overscroll areas? We think no, but yes to touch and trackpad.
   * mouse non-trackpad users also need to access the areas without a mouse. This might require a new visual indicator of overscroll areas similar to resize corner
* If the author does not provide any explicit control buttons, we propose making the container itself a tab stop with added visual affordances that indicate it's activatable.
* Maybe built-in browser shortcuts or default activation (e.g., pressing spacebar or enter while focusing the scroller) to open these areas?

**3. :backdrop and Light Dismiss**
* We plan to support backdrops and light dismiss.
* Does losing focus (e.g., tapping out of the overscroll area) automatically trigger a light dismiss?

**4. Auto-Activation**
* We want to support auto-activation for common gesture patterns like swipe-to-delete or pull-to-refresh.
* If the overscroll element is explicitly a `<button>`, pulling it into view and releasing would automatically click it.
* If the element is a `<dialog>`, releasing the overscroll would implicitly show it.
* Are there accessibility concerns with triggering potentially destructive actions (like delete) via this mechanism?

**Comment by davidsgrogan** [comment]
@vmpstr and @flackr hope to discuss at the regular upcoming May 14th meeting if it's a light agenda and we have time. If not, we can schedule a deep dive or push a week?

**Comment by flackr** [comment]
Just an FYI, the [demos](https://flackr.github.io/web-demos/html/overscrollcontainer/) shown during the meeting are available here:
* Menu: https://flackr.github.io/web-demos/html/overscrollcontainer/menu/
* E-mail: https://flackr.github.io/web-demos/html/overscrollcontainer/email/
* Examples in slides: https://flackr.github.io/web-demos/slides/blinkon21/overscroll/

These can be tested on Chrome Canary 150+ with experimental web platform features enabled.

**Comment by spectranaut** [comment]
Discussed in today's meeting, here are the minutes: https://www.w3.org/2026/05/14-aria-minutes.html#7699
